### ISSUE TITLE
There is no active server

### ISSUE THREAD

**Issue by DamianNurzynski** [tststs/atom-ternjs#299]
I install atom-ternjs plugin to my atom editor. When I want to configure server, I get error "There is no active server" please help :)

**Comment by tanchongshi** [comment]
me too

**Comment by tststs** [comment]
This error only pops up if there is no server running. A server is only running if there is an open text-editor (JavaScript).

- Does the error still show up if there is an active editor?
- if so: which grammar is shown in the bottom right?
- is the server forked for the current directory or are there any error or warning in the console?

**Comment by tanchongshi** [comment]
1: To install a package called language-javasript-jsx
2: Change this grammar to become Babel ES6 Javascript  in the bottom right
3: Successfully solve the problem

😄 thanks

**Comment by anran6688** [comment]
Click on the Save&Restart Server is always the upper right corner of the There is active server or Restarting Server, in the end how to configure, please God

**Comment by camillo777** [comment]
Sorry, but I installed Babel etc but it does not work.
What is this server that is not started?
Thank you!
Camillo

**Comment by valimaggi** [comment]
> Does the error still show up if there is an active editor?

What does this "active editor" mean?

I'm also getting this error  when I select "Configure project". I do it as instructed

- Open any JavaScript file from within your project

- Navigate to Packages -> Atom Ternjs -> Configure project

But the error occurs every time. It's a React project with JSX files (.jsx) and Javascript files (.js). And .css too .

**Comment by valimaggi** [comment]
Sorry, I didn't try what camillo777 said one post above mine. It worked for me. Disabling React-package (+ installing language-babel package) did it for me too. I don't know why.

**Comment by myzhoujuan** [comment]
me too

**Comment by reohjs** [comment]
Hi, @tststs is it possible to use this package with custom grammars? I'm using [language-semantic-js](https://github.com/p-e-w/language-javascript-semantic) 

This might be relevant: https://github.com/p-e-w/language-javascript-semantic/issues/31

**Comment by himanshupathakpwd** [comment]
My case: 

I had installed language-javascript-jsx package [https://atom.io/packages/language-javascript-jsx](url) 

What I did: 
1. Uninstalled language-javascript-jsx package. 
2. Installed lagnuage-babel package [https://atom.io/packages/language-babel](url)
3. Restarted atom

**Worked**

**Comment by kkostov** [comment]
Indeed, I also had `language-javascript-jsx` installed but after uninstalling it and instead using `language-babylon` the tern server was available again.

I believe this issue is a duplicate of #297.

**Comment by Zhang-WW** [comment]
遇到这个问题，在本地项目打开一个js文件，然后在选择配置文件就发现可以了

**Comment by lilingling3** [comment]
i also have the same questions

**Comment by Tapan221** [comment]
As mentioned by Himansu:
Uninstall language-javascript-jsx package.
Installed lagnuage-babel package 
Restart atom.
This worked fine for me as well

**Comment by ashevts** [comment]
> Uninstall language-javascript-jsx package.
>Installed lagnuage-babel package
>Restart atom.
>This worked fine for me as well

.js files:
started to work fine

.html files:
Doesn't works.
Switching grammar in the right bottom corner to babes and various versions of ES doesn't works as well.

**Comment by kcasas** [comment]
Just uninstalling `language-javascript-jsx` and then restarting atom worked for me.

**Comment by scagood** [comment]
#361 Might fix this :eyes:
If the issue that you're facing is when the language scope name changes

**Comment by TrySpace** [comment]
For me, I had not opened a `.js` file yet, only `.ts` or `.tsx` which I had configured in `.tern-project`
